src/context/FhirClientProvider.js

```diff
@@ -55,6 +55,17 @@ export default function FhirClientProvider(props) {
             console.log("Patient loaded.");
             setPatient(result);
             setError(null);
+            writeToLog(
+              "info",
+              ["authSessionCreated"],
```

Comment:
@achen2401  can we make this "authSessionStarted", to be consistent w/ the text here and with https://github.com/uwcirg/asbi-screening-app/pull/70/files#diff-03e5d20a292be4f059c14c3668c59f1e70f2194e03643a212725c00cc840be3fR147 ?